docs/api/courses.md

```diff
@@ -0,0 +1,153 @@
+# `Course`
+A `Course`, also known as `Curricular Unit`, is an academic endeavor composed by classes, lecturers, etc. The full representation of a `Course`, in the [application/vnd.siren+json](https://github.com/kevinswiber/siren) media type.
+
+## Properties
+* `acronym`: the course's unique acronym; an abbreviation of its name
+  - type: **text**
+  - e.g. "DWA"
```

Comment:
Shouldn't it be "DAW" instead of "DWA"?

Reply:
The name of the course is "Development of Web Apps". Therefore, the acronym should be "DWA".